src/go-postgres/devcontainer-template.json

```
@@ -7,19 +7,18 @@
     "publisher": "Dev Container Spec Maintainers",
     "licenseURL": "https://github.com/devcontainers/templates/blob/main/LICENSE",
     "options": {
-		"imageVariant": {
+        "imageVariant": {
             "type": "string",
             "description": "Go version (use -bookworm, or -bullseye variants on local arm64/Apple Silicon):",
```

```suggestion
            "description": "Go version:",
```

I believe it's safe to remove this comment, as there are no other variants options available